NDTensors/Project.toml

```
@@ -1,7 +1,7 @@
 name = "NDTensors"
 uuid = "23ae76d9-e61a-49c4-8f12-3f1a16adf9cf"
 authors = ["Matthew Fishman <mfishman@flatironinstitute.org>"]
-version = "0.3.0"
+version = "0.4.0"
```

```suggestion
version = "0.3.1"
```
I think a minor version bump is ok, the HDF5 functionality of NDTensors is just serving ITensors anyway.

---

I didn't think about it that way. Right – no one should be depending on that part of NDTensors directly.